app/src/main/java/org/kiwix/kiwixmobile/search/SearchActivity.java

```diff
@@ -66,12 +67,14 @@ protected void onCreate(Bundle savedInstanceState) {
     View contentView = LayoutInflater.from(this).inflate(R.layout.search, null);
     setContentView(contentView);
 
+    if (savedInstanceState != null) {
+      searchText = savedInstanceState.getString("text");
```

Comment:
Can we extract "text" to a constant in our Constants file please

Reply:
Will make the changes